src/index.html

```diff
@@ -17,6 +17,86 @@
     />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
```

Comment:
Use semantic tags where possible. Here, consider using the `nav` tag instead of `header` for the navigation menu. It improves your page SEO and helps screen readers.